.github/workflows/on_pr_master.yml

```diff
@@ -114,6 +114,9 @@ jobs:
         consul_acl_default_policy: ["consul_acl_deny"]
         nomad_acl: ["nomad_acl_enabled", "nomad_acl_disabled"]
         hashicorp_binary: ["hashicorp_oss", "hashicorp_enterprise"]
+        nomad_acl_master_token: ["nomad_acl_master_token_random", "nomad_acl_master_token_master"]
+        consul_master_token: ["consul_master_token_random", "consul_master_token_master"]
+        vault_master_token: ["vault_master_token_random", "vault_master_token_master"]
```

Comment:
I think we could change these from three to one, `master` or `random` for all. If we see that we need more granularity we can add that at a later stage, but if something fails it should be easy enough to see what fails from the ansible names

Reply:
> Don't you need to change https://github.com/fredrikhgrelland/terraform-nomad-presto/blob/e0388ef7bdd91004a75b9eb6a455433a4dd774c4/.github/action/create-env.py so that you create a random string for the master token?
> Or is it enough to use something different than `master`?

I think we should use a random generated token. I also suggest we can use the same random token for all services (Nomad, Consul etc). Or WDYT?

In this case you can use `uuid` from Python3 standar lib:
```python3
>>> from uuid import uuid4
>>> id = uuid4().hex
>>> id
'74fcc13f1c964a2795243ad8bdf86f00'
```